config.properties

```diff
@@ -0,0 +1,21 @@
+title = RAINBOW BOOKSTORE
+version = 1.0
```

Comment:
you need to use this property. Maybe in the first screen

Reply:
ok